docs/users/wallets/backup.md

```diff
@@ -0,0 +1,116 @@
+<!--
+order: 4
+-->
+
+# Backup
+
+Below, it's detailed how to backup your wallet with [evmosd](../../validators/quickstart/binary.md).
+
+## Mnemonics
+
+When you create a new key, you'll recieve a mnemonic phrase that can be used to restore that key. Backup the mnemonic phrase:
+
+```bash
+$ evmosd keys add mykey
+{
+  "name": "mykey",
+  "type": "local",
+  "address": "evmos1n253dl2tgyhxjm592p580c38r4dn8023ctv28d",
+  "pubkey": '{"@type":"/ethermint.crypto.v1.ethsecp256k1.PubKey","key":"ArJhve4v5HkLm+F7ViASU/rAGx7YrwU4+XKV2MNJt+Cq"}',
+  "mnemonic": ""
+}
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+banner genuine height east ghost oak toward reflect asset marble else explain foster car nest make van divide twice culture announce shuffle net peanut
+```
+
+To restore the key:
+
+```bash
+$ evmosd keys add mykey-restored --recover
+> Enter your bip39 mnemonic
+banner genuine height east ghost oak toward reflect asset marble else explain foster car nest make van divide twice culture announce shuffle net peanut
+{
+  "name": "mykey-restored",
+  "type": "local",
+  "address": "evmos1n253dl2tgyhxjm592p580c38r4dn8023ctv28d",
+  "pubkey": '{"@type":"/ethermint.crypto.v1.ethsecp256k1.PubKey","key":"ArJhve4v5HkLm+F7ViASU/rAGx7YrwU4+XKV2MNJt+Cq"}'
+}
+
+$ evmosd keys list
```

Comment:
let's split these 2 commands and add a small text like "Verify that your key has been restored using the following command:"